src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -263,6 +271,37 @@ def _create_batch_script(job_dir, task_list_file, n_jobs, tool):
     return batch_script_file
 
 
+def _create_job_dir(root_dir, tool, kbase_collection, load_ver, force=False):
+    """
+    Create the job directory
+
+    Job directory structure: <root_dir>/<TASKFARMER_JOB_DIR>/<kbase_collection>_<load_ver>_<tool>
+    """
+
+    job_dir = os.path.join(root_dir, TASKFARMER_JOB_DIR, f'{kbase_collection}_{load_ver}_{tool}')
+
+    if os.path.exists(job_dir) and force:
+        shutil.rmtree(job_dir)
+
+    os.makedirs(job_dir, exist_ok=True)
+
+    return job_dir
+
+
+def _append_task_info(root_dir, task_info):
+    """
+    Append the task information to the task info file
+    """
+
+    task_info_file = os.path.join(root_dir, TASKFARMER_JOB_DIR, TASK_INFO_FILE)
+
+    # Append the new record to the file
+    with jsonlines.open(task_info_file, mode='a') as writer:
+        if not os.path.exists(task_info_file):
+            writer.write([])
```

Comment:
I'm not understanding why this part is necessary

Reply:
you are right. I don't need this. removed.

Reply:
:+1: